How_to_use.py

```diff
@@ -0,0 +1,20 @@
+#Example how to load dataset by function from module and to show single picture
```

Comment:
How about naming it example.py? Additionally - python files rather begin with lower case:
https://softwareengineering.stackexchange.com/questions/308972/python-file-naming-convention